modules/swagger-codegen/src/main/resources/lua/model.mustache

```diff
@@ -0,0 +1,28 @@
+{{#models}}
+{{#model}}
+{{>partial_header}}
+-- {{classname}} class
+local {{classname}} = {}
+local {{classname}}_mt = {
+  __name = "{{classname}}";
+  __index = {{classname}};
+}
+
+local function cast_{{classname}}(t)
+  return setmetatable(t, {{classname}}_mt)
+end
+
+local function new_{{classname}}({{#vars}}{{name}}{{#hasMore}}, {{/hasMore}}{{/vars}})
```

Comment:
@daurnimator  in the constructor method signature, I've included all parameters (required, optional)